src/TS/services/runInfoDomManipulation.ts

```diff
@@ -41,16 +41,25 @@ export function addLapInfo(
     throw new Error('Seems like an error from our side!');
   }
 
+
   let index = 0;
   for (const lap of lapSummaries) {
     const lapInfoItem = document.createElement('li');
-
+    lapInfoItem.setAttribute(
+      'class',
+      'w-full border-b border-borderColor flex items-center py-2',
+    );
```

Comment:
can also try .className = "...." instead of setAttribute. But it is just preference.